haystack/logging.py

```diff
@@ -55,7 +56,14 @@ def configure_logging(use_json: bool = False) -> None:
     # https://www.structlog.org/en/stable/standard-library.html#rendering-using-structlog-based-formatters-within-logging
     # This means that we use structlog to format the log entries for entries emitted via `logging` and `structlog`.
 
-    use_json = os.getenv(HAYSTACK_LOGGING_USE_JSON_ENV_VAR, "false").lower() == "true" or use_json
+    if use_json is None:  # explicit parameter takes precedence over everything else
+        use_json_env_var = os.getenv(HAYSTACK_LOGGING_USE_JSON_ENV_VAR)
```

Comment:
Should we `lower()` this?

Reply:
yes, also just saw that this breaks for jupyter. one sec

Reply:
done